source/2020-08-25-accessibility-working-group-update.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Accessibility Working Group Update
+author: Melanie Sumner
+tags: Recent Posts, A11y, Accessibility, 2020
+responsive: true
+---
+
+Back in March, an Accessibility (A11y) Strike Team was formed to address the issues outlined in [Ember RFC Issue 595 - Technical Accessibility Issues in New Ember Apps](https://github.com/emberjs/rfcs/issues/595). This blog post is to update the community on that group's work.
+
+The Strike Team met weekly between [March 25](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-03/march-25.md) and [June 24](https://github.com/ember-a11y/core-notes/blob/ember-a11y/ember-a11y/2020-06/june-24.md). We even managed to celebrate [Global Accessibility Awareness Day](https://blog.emberjs.com/2020/05/21/global-accessibility-awareness-day.html)!
+
+READMORE
+
+There were five items on the list: 
+
+1. Application Language Support
+2. Label/Input Support
+3. Page Title Support
+4. Support for `...attributes`
+5. Accessible Routing Support
+
+## Active Work 
+
+### Application Language Support
+
+<!-- alex ignore easy -->
+
+The goal of application language support was to provide a mechanism for new Ember apps to pass [WCAG Success Criteria 3.1.1 - Language of Page](https://www.w3.org/TR/WCAG21/#language-of-page). To meet that goal, [Joseph Sumner](https://github.com/josephdsumner), [Jamie White](https://github.com/jgwhite), [Ava Wroten](https://github.com/hergaiety) and [Melanie Sumner](https://github.com/melsumner) authored [RFC 635](https://emberjs.github.io/rfcs/0635-ember-new-lang.html), which proposed a `--lang` flag for `ember-cli`. This flag would enable developers to declare the application's language while they were creating the app, allowing them to more easily meet this particular success criteria. This RFC was merged on June 12th; the feature itself, authored by [Joseph Sumner](https://github.com/josephdsumner) with help from [Rob Jackson](https://github.com/rwjblue), was merged on August 24th.
```

Comment:
**question:**

I noticed that you added `th` to a date. I wondered if this helps screen readers pronounce the right word.

Reply:
That's an interesting question, but I don't think I know the answer off the top of my head. I'll need to check. 👍🏻